client/ayon_hiero/api/plugin.py

```diff
@@ -648,17 +664,39 @@ class PublishClip:
     rename_default = False
     hierarchy_default = "{_folder_}/{_sequence_}/{_track_}"
     clip_name_default = "shot_{_trackIndex_:0>3}_{_clipIndex_:0>4}"
-    base_product_name_default = "<track_name>"
+    subset_name_default = "<track_name>"
     review_track_default = "< none >"
-    product_type_default = "plate"
+    subset_family_default = "plate"
```

Comment:
These are the wrong names in AYON the `product_name` and `product_type` in the old code are the correct AYON names for these attributes.

Reply:
Yes, this would reintroduce outdated terminology.

Reply:
I've renamed subset for product everywhere.